resources/ui-app/package.json

```diff
@@ -11,27 +11,42 @@
   "scripts": {
     "start": "stripes serve",
     "build": "stripes build --output ./output",
-    "test": "stripes test nightmare --run demo --show",
+    "test": "stripes test karma",
```

Comment:
Could we retain the old nightmare script under `test-int`?  This would remain a placeholder for integration test samples and match that for platforms.  Somewhat related to STCLI-93

Reply:
Added back